plugins/labhub.py

```diff
@@ -239,7 +247,13 @@ def newcomer_difficulty_level(user, iss):
             2. A newcomer asks for assignment to an issue with difficulty higher
                than low.
             """
-            if self.TEAMS[self.GH_ORG_NAME + ' newcomers'].is_member(user):
+            if (self.TEAMS[self.GH_ORG_NAME + ' newcomers'].is_member(user) and
```

Comment:
Yes, isn't that expected, maintainers are eligible to be assigned to issues that are not labelled or are higher than low